### DISCUSSION TITLE
AI and Our Climate Crisis

### DISCUSSION THREAD

**Discussion by TomRiley194505** [daveshap/ACE_Framework#13] [General]
AI development needs to be done in the real world of facing our climate crisis and not in a make-believe world of a Rockwell painting or a computer game.  It is one thing to leave our next generation with a deadly problem, it is another not to even teach them about it.

What we need is a social breakthrough leading to an all-hands-to-the-pumps effort to build a sustainable Earth.  What we got was ChatGPT.  Our task then is to convert the break we got into the break we need.  I think we could make a real contribution to this effort here.

As an example, the movie “Her” is discussed.  In this movie our climate crisis has clearly passed, but no explanation of how is ever given, nor is the effects of that win on society explored.  You can’t get there from here!  Or, at least not without a road map and a lot of AI help.

After I retired from NASA nine years ago, I continued to coach high school students in STEM.  It quickly became apparent that my grassroots space effort, The Big Moon Dig, was counterproductive.  What they immediately needed was a clear vision of the problems they must live through from our climate crisis.  I shifted to writing climate crisis fiction for young people. 
 
That said, all the code languages I have ever written in are dead and gone.  The most I can volunteer for this effort is the review technical papers from a climate crisis viewpoint.  That I would be most happy to do.  Tom Riley

**Comment by daveshap** [comment]
AI in general can help tackle climate crisis in a million different ways. From accelerating science to aiding the deployment of renewables, ACE can help too

**Reply by TomRiley194505** [reply]
Yes, but when faced with a great problem, like our climate crisis, it is best to have at least a secondary team working directly on it.

**Reply by bitplane** [reply]
Climate isn't the core issue, it's part of it but the elephant in the room is the rate of change caused by exponential growth of an economy built on exploiting the natural world. Realistically AI is going to bolster that system, help us eat the planet at a faster rate, directly compete with humans for energy and trample on anyone who interferes with exponentially increasing production. Priorities here ought to be:

1. Protect democracy from being usurped by powerful actors using ML/AI/Big Stats and propaganda. If we don't do this we are turkeys voting for Christmas. Democratising AI is essential to this.
2. Ensure that ordinary humans are economically net positive, so we can focus on things other than immediate survival (i.e. killed by hunger and/or war).  Democratising AI is essential to this too.
3. Solve fusion, so we get enough energy to run ever more technology without perverse economic incentives to burn all the fossil fuel and every piece of organic matter for electricity. An informed population, funding via taxation, and having enough compute to do the job are our best bets here.
4. Actually fix the planet: reduce the birth rate, outlaw the plunder of the wild, hit equilibrium with emissions and environmental change, redesign wealth as something other than "planet loot owed to me"

If we don't do the first 3 things then we are destined to go through a very unpleasant population bottleneck and will not have the luxury to consider the last one.

**Reply by ippolite** [reply]
I don't think that the birth rate belongs to the point of "fixing the planet". 
I'm no sociology expert, but I have recently listened to a lecture by one. Ekaterina Shulman said that the reduction in birth rate is an inherent part of the developing society. Say, when we relied on horses and mules to hoe the fields, we also didn't have the antibiotics and proper understanding of basic medical manipulations (like washing thy hands). So, to ensure that your son would survive long enough to become another worker for the field, you'd have to play a probability game and have 5-6 sons. When we generally moved to cities, an average family didn't need 12 children anymore, they could've settled on 3 or 4. Schools, hospitals, fire departments -- I grew up with everything as if given. Yet, there are countries in the world where families can't have all of their children to be literate.

What I'm trying to say is, the sheer power of AI should be used to reform those societies and bring them up to some standard. By advancing them, we will eliminate the need to have 8-10 children so the lucky one out of them will survive, get financially stable and actually help you in your senior years. This will improve the overall state of the planet because we, humanity, would naturally stop growing around the 11-12 billion mark.

From my own experience, I've done a case on global hunger past week. We have about 828 million hungry people, about 8 billion people in total and we produce enough food (from a calorie standpoint) to feed 10 billion. It's clearly not about the underproduction 

P.S.: I also believe that the plundering of the wild is already illegal and it only helps so much, AI can help us to work out some new approach that might actually work.

**Reply by TomRiley194505** [reply]
Yes, population is critical to working through our climate crisis.  We are currently in severe overshoot and will settle (not collapse!) to the sustainable level for a stressed Earth.  This estimate is about 2.5B over 150 years.  References:  "Empty Planet", Ibbitson and "Overshoot", Catton.
My point here is that we need a major breakthrough so that we can work as one great team on real problems.  What we got was ChatGPT.  Will the work prosed here get us nearer to making the breakthrough we got into the one we need?  I give it a definite maybe.  Thanks Tom Riley

**Reply by bitplane** [reply]
The problem is, technology gives people power, and the more power you have to charge your surroundings the more you can convert them into stuff that's useful for you. So the better technology gets, and the richer people become the more the world will be destroyed. If you look at standards of living, people in the US and Europe use around 100x more power than the median person. 10 billion people having a standard of living that's of using 500kwh/day is not sustainable, we'd need to produce over 100x as much energy and with that power comes 100x as much change to the natural world.

Our civilization is built mostly on mutant grass seed and fossil fuels, with mutant grass seed being about twice as expensive as dead shale. Triple the cost of energy and it makes economic sense for farmers to grow and sell rape instead of corn, rice, oats etc. If GPT4 could do the work of 10 people, they will not be able to work for cheap enough to undercut it while affording food. Banning fossil fuels while nation states are about to go through an AI arms race will put energy prices up enough to the point where famine is guaranteed in poorer nations.

Also it's not natural progression for societies to shift to having fewer children, it's the invention of hormonal birth control, the sociological/cultural impact of mass media and American feminism. The economic effect is to double the workers, reduce family sizes and have a powerful consumer economy. It's a good strategy - give a platform to a masculine-female movement, demonize motherhood (i.e rather than support young mothers in education) and people contribute more to the economy than their families which are now smaller and richer. Brain-draining the cream of competing nation states through immigration to keep the population growing at the expense of the competition is the cherry on top. Highly effective, but if machine intelligence replaces thinking work the system will have to change to something far, far worse. I'm convinced we need AGI agents in the hands of the people or we're doomed.

**Reply by ippolite** [reply]
That premise of 100x change to the environment for 100x more energy does sound like (and I mean it with no offence) a Malthusian trap 2.0. We are on the brink of having fusion power plants (by ~2050) and that is just not what has been known before. Big inventions make big economic and social changes. I think it is safe to say that we should not extend our current trends anywhere after 5 years from now. 
  Let's rewind back to 2018. Could you imagine speaking about autonomous AI agents back then? Or RTX implemented in phones?  If we pick a specific date, that fancy raytracing wasn't even a thing! In 2018 we made 26.750 TW/h of electricity, in 2022 it was 29.165 TW/h. Unfortunately, we didn't build many nuclear power plants, that is true. What is also true is that we didn't have conversations like "Oh no, an AI is taking my job". And for some set of translators, it already did.

  I agree with Dave on our movement to a post-labour economy and a need for universal basic income. If an AI agent can do my job harder-better-faster-stronger, then it might also propose how should we navigate our society through the transitional phase. 
_Maybe, a specific agent should be built to consult the governments around the world?_

P.S.: If having more children is natural, then how come Australia isn't? Pretty far away from wars, and quite rich, yet there's definitely been a downward trend in the average household size for the last 100 years. The population went x5, yet households shrank from 4.5 to 2.5. I'm willingly omitting here the biological consequences for women from frequent births. 

I also don't understand the part about "competing nation states". I can't see a reason how enabling the movement of people can tamper the competition between them.

Honestly, I don't feel like continuing my contribution to this thread. It feels like it's too far off the ACE topic

**Reply by bitplane** [reply]
> Unfortunately, we didn't build many nuclear power plants, that is true.

They're probably not the best idea at the moment. When you build a plant you're making a bet on long term stability, so knowing what we know, it's unwise to bet the region that you, your children and your grandchildren can keep it safe from war, natural disaster, solar flares, neglect, poverty, administrative corruption, social collapse, terrorism and future technological unknowns until it's decommissioned. The US might be safe enough bar flares (5% chance), but France are crazy to rely on them given their record for being invaded in the past century. Bangladesh and Iran?! Whoever sold them the plants didn't get paid in reality cheques!

> Let's rewind back to 2018. Could you imagine speaking about autonomous AI agents back then?

Pretty sure I was. I was expecting gaming to lead the charge rather than search engines and chip companies though, and it is surprising how much natural language has unlocked and how the transformer has utterly dominated the space. But we were expecting human-level intelligence by 2025, just didn't know how it'd manifest.

>  If an AI agent can do my job harder-better-faster-stronger, then it might also propose how should we navigate our society through the transitional phase.

Well by default we go through a population bottleneck! The most powerful AI "advisors" will serve the most powerful actors and work against other AIs in a squabble for power. Nation states reign as they have a monopoly on violence and legislative power, but democracy means that power is mailable to agents that are good at prediction and manipulation. If we're lucky they'll be on our side for long enough for us to die naturally or merge with tech. But if the game becomes too complex to be played by the resources we have (agents in the same realm of intelligence as the major players),we'll lose our grip on that monopoly and the pointy end will bend towards us. If living on a UBI reservation comes with the promise of comfort and dying of old age then I'll take it, but could you trust a promise made by amoral agents?

> If having more children is natural, then how come Australia isn't?

Same reason as the rest of the West. Hormonal birth control and social/cultural pressure. The key indicator for fecundity is the age of a woman when she has her first child; population decline is a side effect of delaying adulthood until the age of 30. Without education and wealth as a way to suppress motherhood, odds are we'll see some other population control mechanism come into play. Hopefully not one that spells out what the true brutality of "post-work" really means.

**Reply by TomRiley194505** [reply]
We needed fission reactors to make the A-Bomb.  We did not need fusion reactors to make an H-bomb.  Fission yes; fusion no.

Our climate crisis is full of non-linear elements (called tipping points).  These make perditions iffy at best. The same goes for technology.  If these two fields of non-linearities cancel each other out, it will be nothing short of a miracle.  Arguing over which will hit first is a fool's errand. 

The population of the Earth is now in server over-shut.  Imagine a robot arm movement with the gain turned way up.  I will peak this century and then settle (not collapse!) to the sustainable level of a stressed Earth.  Current estimates are around 2.5 billion.  The driver is urbanization.  City women have fewer children.  This effect is currently on autopilot.  There is nothing we need do to cause it; nothing we can do to stop it.  "Rejoice, rejoice, you have no choice."
Enjoy, Tom Riley

**Comment by TomRiley194505** [comment]
One of the main characters in my novels is an AI.  It is an AI so close to the agent called for in the first task here that I find I have scared myself.

JanetO is the AI member of an investigations team.  She is smart and personable.  She has no robotic body (as this would have an enormous carbon footprint) but has a human image that can jump from screen to screen as needed.  She is smart and a wise ass. She might give people a clearer idea of what is proposed here and the consequences.

The first two books are:
	“Born to Storms” by Tom Riley, a YA coming of age story
	“A Climate of Revenge” by Tom Riley, a murder mystery.
The third book “Dark Heat” is complete but is now languishing for lack of a literary agent.  This time ChatGPT-4 was a member of this writing team.  Thank you, Tom Riley

**Reply by TomRiley194505** [reply]
I have about five paperback review copies on hand.   Please let me know if you would like on.  Please say if you have young people in your household so I can include the Young Adult too.  Leave a rely here so we can work out the shipping address.

**Reply by daveshap** [reply]
Let's tone down self-promotion, please. This is not a good use of signal-to-noise ratio. Please review the C3P0 community policy. https://github.com/daveshap/C3P0

**Reply by TomRiley194505** [reply]
OK

**Comment by daveshap** [comment]
Relevant: https://youtu.be/RNhbqQefPSg?si=FCcpekgDu0lL58pS Can AI Help Solve the Climate Crisis? | Sims Witherspoon | TED

**Reply by TomRiley194505** [reply]
Yes, this is a small positive effort.  One problem is that AIs trained on past data may not make good choices if the overall environment is shifting as it now is with climate.

**Comment by TomRiley194505** [comment]
Gato and Mark 14 Story

What does the GATO concept used here have to do with submarines and what is the warning against pulling a Mark 14?  
A Gato is a tropical fish nobody has ever heard of.  American WW2 submarines were named after fish.  The first ship in a series names the class.  Consequently, the Gato Class were 70 of the best submarines of WW2.

For the first half of WW2 our efforts in the Pacific were let down by a truly terrible torpedo, the Mark 14. It had at least five major flaws. Not one in ten would sink a ship.

The problem was that the Mark 14 was designed between the wars.  Only 2 or 3 were allocated for testing because of budget limitations.  That simply was not enough. 

When WW2 did break out, the men in charge of the design and testing were promoted to high rank over the Pacific Fleet.  They simply would not hear of problems with their torpedo and blamed the problems on the brave captains.

It took about two years of losing badly before non-interested parties took over the testing and worked out the mirid problems.  Then we started winning.

The lesson here is that limiting testing in time and cost is an invitation to disaster.  And people who have an emotional stake in the project cannot be allowed to limit such testing.  

This is a good story.  Maybe this track needs to be about stories.  Thank you, Tom Riley

**Comment by TomRiley194505** [comment]
AIs Story

For as long as there have been civilizations, there have been grand stories.  From Homer to the Bible, these stories defined what is good and what is heroic in the society.  

The defining AI story today is “Terminator”.  This looks like a disaster waiting to happen and a pressure point worthy of early attention.

We need a new AI story and AIs are good with language.  One of the first tasks of our new AI should then be to write its own defining story.

AIs need structure to write long.  We could suggest “the Snowflake Method”, or “Save the Cat”, but this time we need to pull out all the stops.  We need “A Hero’s Journey” from Joseph Campbell.  It is a complex and intricate structure, but it worked for “Star Wars”; it can work for a new AI now.

In this structure, a complete novice starts out with an unwanted challenge and through seven labors, (it is always seven), becomes Obi-Wan Kenobi, a Servant Leader.  The Servant Leader can then provide leadership to all who follow and does so with knowledge won honestly.

A few of these labors can be defined by our climate crisis.  I will work up one as an example related to saving our seas next.  Enjoy, Tom Riley

**Comment by TomRiley194505** [comment]
Labor of the Sea 			(1436 words)

So we have a novice AI and we need to send it out, like Hercules, on seven labors where it may become wise through its own experiences.  The following labor is designed to aid Earth’s oceans in recovery from the degradations of our climate crisis. 

Here are suggestions for a structure that the AI must first develop and then deploy into a fictional account.  This involves (1) identifying possible solutions and their dangers, (2) proposing an approach that an AI could participate in, and (3) detailing a great story of this solution in action.

The problem is multi-fold, including:  (1) excess CO2 in the atmosphere is acting like a blanket and increasing the overall temperature, (2) the excess CO2 is going into solution making the seas more acidic, and there are many other less appreciated problems.  

There have been proposed a number of activities to address these problems, including:
1.	Adding iron to the ocean to precipitate the carbon
2.	Enhancing clouds to block sunlight
3.	TBD (To Be Determined)
A number of these efforts have been tried at small and medium scale but these efforts were not properly monitored and generated more controversy than results.  The specific labor here is then to develop a monitoring system that is up-close-and-personal and can provide the hard data needed to evaluate any ocean based proposal.  All the while, this solution must both be doable and make a great story.

Full scale research vessels are very expensive to both build and operate.  A fleet of Sir David Attenborough class vessels would be both prohibitively expensive and too slow to build.  That said, there are a large number of deep sea capable yachts already built and they are likely to be a-drug-on-the-market as the climate crisis drags down the economy.  Our AI must survey this surplus stock and determine which ones can be economically modified to serve a monitoring function.  This must include the instrumentation, crew requirements, and safety capabilities.  A hundred such vessels should match the need.
Only then can the story begin.  The AI is the exec of one of these many vessels and lives in the ship’s advanced digital system.  At first the voyage goes well.  The vessel has duties to monitor surface waters where iron has been added, to trail in the wake of large container ships that have cloud making equipment on board, to take ground truth data as satellites pass over, and to measure the extend of a garbage patch.  

But this is a stress test.  The AI’s vessel must then take readings much closer to the northern ice than is safe, run ahead of great storms, and fight off pirates who try to commandeer the ship to use for smuggling.  

Whether or not the AI gains any wisdom from this labor will have to be tested when its vessel is save back in port.  Also whether such stories can help replace “Terminator” as the dominate AI tale, and whether they build support for climate crisis efforts, must be independently judged.

Please leave a comment if you think this idea is worth developing.  Enjoy, Tom Riley

**Comment by ScottzCodez** [comment]
The whole climate Crisis is political garbage - lets keep that nonsense out of this forum.  
Good issues to keep = Antipollution, Clean energy, clean rivers, oceans, etc    Keep the good stuff that are common sense and existing before the "Climate Crisis" "Climate Change" political nonsense that's designed to take freedoms, suppress society, tax us more.
The globalists agenda type "Climate Crisis" lock downs that they are talking about, and many other things under the "Climate" umbrella are garbage, ignorant leftist, political junk.

**Reply by TomRiley194505** [reply]
The science under our climate crisis is solid.  This crisis is now coming on faster than expected, especially in the artic.  Planning ahead is almost certain to reduce the problems people will face.  Here, the applications of AI to these problems warrants detailed technical discussion.  We need not be completely blind-sided.

**Reply by daveshap** [reply]
Scott has been banned for trolling, misinformation, and harassment. I enforce a strict zero tolerance policy to wasting time. https://github.com/daveshap/C3P0
